eng/tools/RepoTasks/FileUtilities.cs

```diff
@@ -7,41 +7,42 @@
 using System.IO;
 using System.Reflection;
 
-namespace Microsoft.DotNet.Build.Tasks
+namespace RepoTasks
```

Comment:
Changes here also seem similar to https://github.com/dotnet/core-setup/blob/5cfc2cab25bfc431ef89572aa2705725a95aca32/tools-local/tasks/FileUtilities.cs. That's the intention right?

Reply:
https://github.com/dotnet/arcade/issues/2704

Reply:
Yes, eventually this will be shared.